.env.sample

```diff
@@ -10,7 +10,7 @@ ENABLE_QR_VENUES=true
 CONFIG_VAR_PREFIX=
 NODE_ENV=development
 LOG_LEVEL=debug
-CORS_ORIGIN=
+CORS_ORIGIN=true
```

Comment:
this needs to be a valid value for a real server to be able to start up. empty string is not a valid value